packages/core/jest.config.js

```diff
@@ -11,6 +11,8 @@ const config = {
   moduleNameMapper: {
     '^#src/(.*)\\.js(x)?$': '<rootDir>/build/$1',
     '^(chalk|inquirer)$': '<rootDir>/../shared/lib/esm/module-proxy.js',
+    // Map the connector-kit to the installed version rather than finding it from the `shared` package
```

Comment:
```suggestion
    // Map the connector-kit to the installed version rather than finding it from the `shared` package (which is the default behavior of `mockEsm` in the `shared` package)
```